Rakefile

```diff
@@ -21,7 +21,7 @@ end
 BUILDDIR = File.expand_path(ENV['BUILDDIR'] || '_build')
 PKGDIR = ENV['PKGDIR'] || File.expand_path('pkg')
 FOREMAN_MODULES_DIR = File.expand_path(ENV['FOREMAN_MODULES_DIR'] || '/usr/share/foreman-installer/modules')
-FOREMAN_BRANCH = ENV['FOREMAN_BRANCH'] || 'develop'
+FOREMAN_BRANCH = '1.19-stable'
```

Comment:
Maybe keep the `ENV` option but just replace the default?

Reply:
I don't see value in the ENV option to be honest. Let's lock it down.

Reply:
I suppose some other distro of this could use it. Its put back.